### ISSUE TITLE
HUD only accounts for vertical head movement

### ISSUE THREAD

**Issue by Rudolf339** [swampmo/Jaguar#64]
The HUD symbology does move up/down with the viewpoint, but not every part. The AoA, vertical speed tapes, the heading tape and the selected station / ammo count symbols don't move. 
Also none of the symbology account for left/right movement.

**Comment by NikolaiVChr** [comment]
Yes, parallax is not implemented at all. It just make sure the pitch ladder and horizon fit the eye point.

**Comment by Zaretto** [comment]
This would be a future improvement if we ever get around to it.
